vortex-array/src/array/list/compute/mod.rs

```diff
@@ -48,3 +52,10 @@ impl SliceFn<ListArray> for ListEncoding {
         .into_array())
     }
 }
+
+impl MinMaxFn<ListArray> for ListEncoding {
+    fn min_max(&self, _array: &ListArray) -> VortexResult<Option<MinMaxResult>> {
+        // TODO(joe): Implement list min max
```

Comment:
Remove the impl

Reply:
i went for the pub fn returning a not impl error and impls returning null, would you prefer the pub fn to return non is the compute fn is absent?